### DISCUSSION TITLE
Duplicata

### DISCUSSION THREAD

**Discussion by HydrelioxGitHub** [alexbelgium/hassio-addons#1187] [General]
If I'm not making a mistake, I ve seen that --arr software are now available in the community add-on. What is the best choice? Stick with those in Alex repository or migrate ?

Thanks for sharing your opinions!

**Comment by alexbelgium** [comment]
You should migrate for add-ons in the community store, they'll be better supported! 

This repo is to fix gaps in apps available, but I have zero issue with migration ;) in an ideal world this repo would not exist and everything would be supported by the HA team! Over the years HA has progressively closed the gaps I was fulfilling : they now support SMB mountings ; they now support common config paths (it was even nearly implemented with the same name I was using ;-) ) ; updates are much more regular... So indeed I fully recommend to move to official addons

Only thing is that they haven't implemented ingress on theirs, and don't seem to be open to seeing the disclaimer. Well it's quite doable so I guess it's only a matter of time

**Reply by HydrelioxGitHub** [reply]
I've tried migrate Radarr to the community repo using the backup and restore feature. It worked like a charm, path to medias was identical so I did not even have to change this.

**Comment by HydrelioxGitHub** [comment]
Thanks for your answer. As these add-on are "official" now, will you, in a near future, stop support/update them ?

**Comment by alexbelgium** [comment]
Well you gave me the idea that I should :-) Actually it makes sense - better for me to focus on addons not officially supported. Therefore from the next version there will be a banner appearing encouraging people to migrate to the "official community versions".

Automated updates will still occur, but I'll likely reduce my involvment supporting them.

I am very humbled and glad of the number of people who are using my addons, but in the end everyone benefits to migrating to officially supported addons

**Comment by nstrelow** [comment]
Very cool news, but I do like having Ingress mode.
Frenck kinda added a disclaimer that it would not be feasible to support Ingress (lot of work and can break easily)

I would love to keep using yours if I can, for Ingress :)

Also if possible, can you add to the next changelog that migrating is recommended, nut ingress isn't working? 😁

**Comment by Stainy3244** [comment]
I'm also keeping yours as there's no external drive mount on the community store version. Thanks for all you do

**Reply by alexbelgium** [reply]
I've set the deprecated sign on the *arr add-ons so people migrate to official community repo but actually i'll keep those addons active on my repo with the same update frequency as before ;)

the only add-on I've truly deprecated is vaultwarden as now that the official add-on is regularly updated mine serves no purposes!
